librz/core/cmd_eval.c

```diff
@@ -660,3 +667,135 @@ RZ_IPI int rz_cmd_eval(void *data, const char *input) {
 	}
 	return 0;
 }
+
+RZ_IPI RzCmdStatus rz_eval_getset_handler(RzCore *core, int argc, const char **argv) {
+	if (argc == 1) {
+		rz_config_list (core->config, NULL, 0);
+		return RZ_CMD_STATUS_OK;
+	}
+
+	int i;
+	for (i = 1; i < argc; i++) {
+		RzList *l = rz_str_split_duplist_n (argv[i], "=", 2, false);
+		if (!l) {
+			return RZ_CMD_STATUS_ERROR;
+		}
+		size_t llen = rz_list_length (l);
+		if (!llen) {
+			return RZ_CMD_STATUS_ERROR;
+		}
+		char *key = rz_list_get_n (l, 0);
```

Comment:
check if `key` is empty and print a warning. Moreover, suggest the correct syntax without spaces.